react-demo/package.json

```diff
@@ -15,13 +15,16 @@
     "highcharts": "^6.2.0",
     "lodash.clonedeep": "^4.5.0",
     "react": "^16.9.0",
-    "react-dom": "^16.9.0",
     "react-app-polyfill": "^1.0.2",
+    "react-dom": "^16.9.0",
     "react-highcharts": "^16.0.2",
     "react-scripts": "3.1.1",
     "typeface-open-sans": "^0.0.75",
     "typescript": "3.6.3"
   },
+  "resolutions": {
```

Comment:
What is this?

Reply:
Whenever I did a yarn upgrade, the react build started failing because create-react-app/eslint-config-react-app@5.0.2 (latest) has a an open defect.  Since we're not using that dependency directly, I used the resolutions tag to specify that I want the previous version@5.0.1.  When the defect is closed we should be able to delete this tag.

https://github.com/facebook/create-react-app/issues/7712